exercises/concept/parsing-log-files/.docs/instructions.md

```diff
@@ -82,7 +82,7 @@ lp.RemoveEndOfLineText("[INF] end-of-line23033 Network Failure end-of-line27");
 
 ## 5. List lines with extremely weak passwords so the guilty can be punished
 
-Before expunging the passwords from the file we need to list any instances where passwords begin with the text "password".
+Before expunging the passwords from the file we need to list any instances where the contents begin with the text "password" followed by any alphanumerics in the same word; i.e. an exact match of "password" is not flagged, but "passwordsecret" is - see example.
```

Comment:
I completely agree that this needs some improvement. I'm not sure the above change fully captures what we want to do with the method, namely to use the `\s` and `\S` character classes: https://docs.microsoft.com/en-us/dotnet/standard/base-types/character-classes-in-regular-expressions#whitespace-character-s

If you check the [exemplar implementation](https://github.com/exercism/csharp/blob/f89b5cc5fd09e24ca0d90f435b5f387c326484ba/exercises/concept/parsing-log-files/.meta/Exemplar.cs#L35), you'll see the code that we want a student to write (or something very similar).
[The tests](https://github.com/exercism/csharp/blob/f89b5cc5fd09e24ca0d90f435b5f387c326484ba/exercises/concept/parsing-log-files/ParsingLogFilesTests.cs#L67-L84) are fairly basic though, so I can see where the confusion is.
What we want is to check if a line contains a password that is weak, which means that the password:

- is preceded by a white-space character
- starts with the text "password"
- followed by one or more non-whitespace characters

Could you perhaps rephrase the entire section, as the following line is just wrong:

> Lines containing an offending password should be returned prefixed with ": ".

As can be seen in the example:

```csharp
var lp = new LogParser();
lp.ListLinesWithPasswords(new string[] {"my passwordsecret is great"});
// => "passwordsecret: my passwordsecret is great"
lp.ListLinesWithPasswords(new string[] {"my password secret"});
// => {"--------: my password secret"}
```

We want to return the actual password followed by a `:` if it is weak.

Reply:
@ErikSchierboom I've updated the instructions, removed the references to "password" in Task 5 to avoid the confusion above^, and updated the Test Cases to align with the instructions a bit more. 

I think the "gotcha" someone would have going through this is figuring out the `\s` and `\S` which was missing from the previous setup.

E.g. In [my first pass](https://github.com/mdrakiburrahman/exercism_dotnet/blob/509c9e4a43843cead6c9f2fb02a1e9f4ddb52ca9/parsing-log-files/ParsingLogFiles.cs#L64) of the solution I had no idea I was supposed to use `\s` and `\S` from the Test Cases/instructions.

Lmk if you think of further changes.